### ISSUE TITLE
Set keybinding for non-commands

### ISSUE THREAD

**Issue by Perry3D** [FeiyouG/commander.nvim#3]
Hi,

thank you for this great plugin.

What do you think about adding keybindings for non-commands (just key sequences).
For example i have this keybinding in my config:
```lua
utils.map('n', '<leader>a', '"ayiw')
```

Would be nice to also add this into the command center.

/edit: This would also enable multiple commands. For example
```lua
utils.map('n', '<F5>', ':CMake build<CR>:copen<CR>')
```

**Comment by FeiyouG** [comment]
This is definitely on the roadmap!

**Comment by FeiyouG** [comment]
This feature is implemented in ecb8b488584807bd14eddb1443b0797e3033bdcc
